process/smartContract/builtInFunctions/esdtFreezeWipe.go

```diff
@@ -85,6 +85,20 @@ func (e *esdtFreezeWipe) ProcessBuiltinFunction(
 	return vmOutput, nil
 }
 
+func (e *esdtFreezeWipe) wipeIfApplicable(acntDst state.UserAccountHandler, tokenKey []byte) error {
+	tokenData, err := getESDTDataFromKey(acntDst, tokenKey, e.marshalizer)
+	if err != nil {
+		return err
+	}
+
+	esdtUserMetadata := ESDTUserMetadataFromBytes(tokenData.Properties)
+	if !esdtUserMetadata.Frozen {
+		return process.ErrCannotWipeAccountNotFrozen
+	}
+
+	return acntDst.DataTrieTracker().SaveKeyValue(tokenKey, nil)
```

Comment:
we should resolve to not send back this data to the metachain -> does we can have a silent error here. with returnMessage -> cannot wipe as it was not freezed.

Reply:
plus add code in ProcessIferror to not Add the created SCR if shardID := sc.shardCoordinator.ComputeId(scr.GetRcvAddr())
if shardID == core.MetachainShardId && scr.GetGasLimit() == 0 && scr.GetValue().Cmp(zero) == 0 {
   continue
}

Reply:
my suggestion does not work - it can remain as it is. it works fine.